kie-remote/kie-remote-services/src/main/java/org/kie/remote/services/rest/jaxb/DynamicJaxbContext.java

```diff
@@ -338,7 +343,7 @@ private void removeClassFromJaxbContextClassList( List<Class> classList, Illegal
                String className = classLocs.listIterator(classLocs.size()).previous().toString();
                Class removeClass = null;
                try {
-                   removeClass = Class.forName(className);
+                   removeClass = projectClassLoader.loadClass(className);
```

Comment:
@mrietveld @mswiderski the main difference is really that Class.forName(...) initializes the loaded class. We did have problems a couple years back because we used ClassLoader.loadClass(...) and we had to replace them by Class.forName(...), but I don't recall the actual scenario. I default to pretty much always use Class.forName(...) ever since, but @mrietveld seems to have a reason not to? Here are a couple articles about it:
http://www.javaworld.com/article/2077332/core-java/get-a-load-of-that-name.html
http://stackoverflow.com/questions/8100376/class-forname-vs-classloader-loadclass-which-to-use-for-dynamic-loading